spec/services/measurements_creator_spec.rb

```diff
@@ -0,0 +1,60 @@
+require "spec_helper"
+
+describe MeasurementsCreator do
+  let(:measurements_attributes) { [{:longitude=>25.4356212, :latitude=>56.4523456, :time=>"2016-05-11T17:09:02", :milliseconds=>925, :measured_value=>59.15683475380729, :value=>59.15683475380729}] }
```

Comment:
Please consider one key/value per line so that code does get to horizontal.
Also, consider using the newer syntax (ie `{ key: value }`)

Reply:
I'd suggest you move this in a private method at the end. The attributes don't really matter so it's better "hidden" at the end